src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -24264,6 +24337,541 @@ static public void M3(int x) {}
             );
     }
 
+    [Fact]
+    public void MemberNameAndSignatureConflict_20_ReceiverType_TupleNameDifference()
+    {
+        var src = """
+public static class Extensions
+{
+    extension((int a, int b) receiver)
+    {
+        void M1() {}
+    }
+
+    extension((int c, int d))
+    {
+        static void M1() {}
+    }
+
+    extension(int receiver)
+    {
+        void M1() {}
+    }
+}
+""";
+        var comp = CreateCompilation(src);
+
+        comp.VerifyDiagnostics(
+            // (10,21): error CS0111: Type 'Extensions' already defines a member called 'M1' with the same parameter types
+            //         static void M1() {}
```

Comment:
Consider adding marker comments like `// 1` (or perhaps `#line` pragmas) otherwise it's not very clear which line this diagnostic is reported at. Applies also to similar cases in new tests.

Reply:
Usually the source on the error lines is unique and/or the test scenarios are pretty small.